nfv_tempest_plugin/tests/scenario/test_nfv_advanced_usecases.py

```diff
@@ -150,9 +150,13 @@ def test_numa_aware_vswitch(self, test='numa_aware_vswitch'):
 
         if (not CONF.nfv_plugin_options.target_hypervisor
                 and CONF.nfv_plugin_options.run_live_migration):
-            migrate_kw_args = {'block_migration': True, 'force': True}
+            migrate_kw_args = {
+                'block_migration': CONF.nfv_plugin_options.live_migration_block,
+                'force': True
+            }
             # on 16.X microversion is 2.72
-            if float(CONF.compute.min_microversion) > 2.67:
+            if (float(CONF.compute.min_microversion) > 2.67 and
+                CONF.nfv_plugin_options.nfv_plugin_options.live_migration_block):
```

Comment:
usually flake8 complains on E129 in this case. I would move the line 4 spaces right.

Reply:
flake8 issues solved.